src/excalidraw-embed/get-image.js

```diff
@@ -0,0 +1,29 @@
+const puppeteer = require("puppeteer");
+
+module.exports = async (url) => {
+  // disable sandbox on build env
+  const browser = await puppeteer.launch({
+    args: process.env.NOW_BUILDER ? ["--no-sandbox"] : [],
+  });
+  const page = await browser.newPage();
+  await page.goto(url);
+  await page.click("[aria-label=Export]");
+  await page.click("[aria-label='Scale 3 x']");
+  const frame = await page.mainFrame();
+  const result = await frame.evaluate(
+    () =>
+      new Promise((resolve) => {
+        const button = document.querySelector('[aria-label="Export to SVG"]');
+        const link = document.createElement("a");
+        delete window.chooseFileSystemEntries;
+        document.createElement = () => link;
+        link.click = () =>
+          fetch(link.href)
```

Comment:
This is failing with "TypeError: Failed to fetch" where link.href is `blob:https://excalidraw.com/ab7b2f84-9207-4ee5-83c5-eaa94c0f969c`